BLE_GattServer_FOTAService/MCUboot/target/bootloader/enc_key.c

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
This should be generated. If the repo contains a test key, document how it was generated.

We also need a licence and a warning explaining that this key is not suitable for production use